examples/energy-management-app/energy-management-common/energy-management-app.matter

```diff
@@ -1583,7 +1594,7 @@ provisional cluster DeviceEnergyManagement = 152 {
   readonly attribute ESAStateEnum ESAState = 2;
   readonly attribute power_mw absMinPower = 3;
   readonly attribute power_mw absMaxPower = 4;
-  readonly attribute optional nullable PowerAdjustStruct powerAdjustmentCapability[] = 5;
+  readonly attribute optional nullable PowerAdjustCapabilityStruct powerAdjustmentCapability = 5;
```

Comment:
this changed the type of a structure. This is not backwards compatible.

What is the history here? 

Also API-wise several changes here are not backwards-compatible. Is this still under development? If this was already released we cannot change it anymore.

Reply:
This has not been released yet (it didn't go into 1.3 SVE) and is only provisional/ok to change. It is now in the Matter 1.4 Spec 0.7 ballot in the changed format above.